Makefile

```diff
@@ -1,37 +1,75 @@
+include .bingo/Variables.mk
 SHELL=/bin/bash
 
-GODIRS_NOVENDOR   ?= $(shell go list ./... | grep -v /vendor/)
+PROVIDER_MODULES ?= $(shell ls -d $(PWD)/providers/*)
+MODULES          ?= $(PROVIDER_MODULES) $(PWD)/
+
 GOFILES_NOVENDOR  ?= $(shell find . -path ./vendor -prune -o -name '*.go' -print)
 
-GOPATH            ?= $(shell go env GOPATH)
 GOBIN             ?= $(firstword $(subst :, ,${GOPATH}))/bin
+
 GO111MODULE       ?= on
 export GO111MODULE
 GOPROXY           ?= https://proxy.golang.org
 export GOPROXY
 
-# TODO(bwplotka): Auto install using separate go.mod for tools.
-GOIMPORTS         ?= $(GOBIN)/goimports
+.PHONY: fmt
+fmt:
+	@echo "Running fmt for all modules: ${MODULES}"
+	@$(GOIMPORTS) -local github.com/grpc-ecosystem/go-grpc-middleware/v2 -w $(GOFILES_NOVENDOR)
+	@$(foreach dir,$(MODULES),$(MAKE) fmt_module DIR=$(dir))
 
-all: vet fmt test
+.PHONY: fmt_module
+fmt_module:
+	go fmt $(shell cd $(DIR) && go list ./... | grep -v /vendor/)
 
-fmt:
-	go fmt $(GODIRS_NOVENDOR)
-	@$(GOIMPORTS) -local github.com/grpc-ecosystem/go-grpc-middleware/v2 -w ${GOFILES_NOVENDOR}
-
-vet:
-	# Do not check lostcancel, they are intentional.
-	# TODO(bwplotka): Explain why intentional.
-	go vet -lostcancel=false $(GODIRS_NOVENDOR)
-
-proto: ./grpctesting/testpb/test.proto
-	@scripts/genproto.sh
-
-# TODO(bwplotka): This depends on test_proto, but CI does not have it, so let's skip it for now.
-test: vet
-	cd ./providers/kit/ && go test -v -race ./...
-	cd ./providers/logrus/ && go test -v -race ./...
-	cd ./providers/zap/ && go test -v -race ./...
+.PHONY: proto
+proto: $(PROTOC_GEN_GOGOFAST) $(GOIMPORTS) $(PROTOC) ./grpctesting/testpb/test.proto
```

Comment:
Unrelated to this PR, but we should be able to switch to `google.golang.org/protobuf` now.